### PR TITLE
Plugins: Add Action Links to Plugin Detail View

### LINKED ISSUE #82: Jetpack Plugins: add settings link
Add a settings link to each plugin card, using the new Button component.

1bef9d74-65f5-11e5-9fc7-a63cb6cf1aed

Before we can expect that link that work, we need to make sure we catch each plugin. That could be tricky as it really depends on when and how the plugin decides to add the url to the plugins filter. Currently there is no standard way of doing this.

It might be easier to open a ticket in Core first, and suggest a new filter/function that would allow plugin devs to define where the canonical settings URL when the plugin includes one.

### PR DESCRIPTION
**Pull request by roundhill**
Adds plugin `action links` buttons to the plugin detail view. The plugin developer adds these using the [action link filter](https://codex.wordpress.org/Plugin_API/Filter_Reference/plugin_action_links_(plugin_file_name)).

**To test**
* Ensure your Jetpack site is running 4.7.
* Install some plugins via calypso that use action links (and some that don't). Jetpack and WooCommerce are good examples.
* After installing the plugin, you should see the action links appear as buttons underneath the plugin icon.
* Test the buttons to make sure they link properly to the content.
* Remove the plugin. The buttons should disappear.

Screenshot:
screen shot 2017-02-28 at 3 27 23 pm

Fixes #82

### CODE REVIEW

**Comment by dmsnell** [comment]
code looks good to me @roundhill but I haven't tested it yet

**Comment by roundhill** [comment]
Note: You can now test this with production Jetpack (4.7). I updated the testing instructions.

File: client/my-sites/plugins/plugin-meta/index.jsx [hunk 1]
```diff
@@ -342,6 +344,15 @@ export default React.createClass( {
 							<div className="plugin-meta__meta">
 								{ this.renderAuthorUrl() }
 							</div>
+							{ ! isEmpty( actionLinks ) &&
+								<div className="plugin-meta__action-links">
+									{ Object.keys( actionLinks ).map( linkTitle => (
+										<Button compact icon href={ actionLinks[ linkTitle ] } target="_blank">
```
Comment (vindl) [inline client/my-sites/plugins/plugin-meta/index.jsx]: I think we should add `rel="noopener noreferrer"` here. See https://github.com/Automattic/wp-calypso/pull/7680.

Comment (roundhill) [inline client/my-sites/plugins/plugin-meta/index.jsx]: Thanks, I wasn't aware of that. Added in 5f046dd

**Pull request merged**